charts/internal/gvisor/templates/service-account-gvisor.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: gvisor
```

Comment:
Multiple gvisor installations for different worker pools will all reconciles/overwrite this object, right?

Reply:
see above.